client/src/components/App.jsx

```diff
@@ -1,20 +1,112 @@
 import React from 'react';
+import axios from 'axios';
+import TabNav from './TabNav';
+import TopNav from './TopNav';
+import Cohort from './Cohort';
+import TeamList from './TeamList';
 
 export default class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      initial: [],
+      sprintCohorts: [
+        {
+          name: 'RPT10',
+        },
+        {
+          name: 'RPT11',
+        },
+      ],
+      teamCohorts: [
+        {
+          name: 'RPT07',
+        },
+        {
+          name: 'RPT08',
+        },
+      ],
+      display: 'sprints',
+      selectedCohort: 'RPT10',
+      loading: false,
+      showSegment: false,
+      currentCommitData: {},
+      currentProjectData: {},
     };
+    this.handleSelectCohort = this.handleSelectCohort.bind(this);
+    this.handleSelectDisplay = this.handleSelectDisplay.bind(this);
+    this.handleRepoSelect = this.handleRepoSelect.bind(this);
+    this.checkSprints = this.checkSprints.bind(this);
+    this.checkProjects = this.checkProjects.bind(this);
+  }
+
+  handleSelectDisplay(type) {
+    const selectedCohort = type === 'sprints' ? 'RPT10' : 'RPT07';
```

Comment:
More technical debt -- hard coding cohort IDs.  Let's get this on a todo list for future iteration.